lstchain/scripts/lstchain_mc_dl1ab.py

```diff
@@ -143,11 +143,18 @@ def main():
                     dl1_container.length = length
                     dl1_container.r = np.sqrt(dl1_container.x ** 2 + dl1_container.y ** 2)
 
-                    for p in parameters_to_update:
-                        params[ii][p] = Quantity(dl1_container[p]).value
                 else:
-                    for p in parameters_to_update:
-                        params[ii][p] = 0
+                    # for consistency with r0_to_dl1.py:
+                    for key in dl1_container.keys():
+                        dl1_container[key] = \
+                            u.Quantity(0, dl1_container.fields[key].unit)
+
+                    dl1_container.width = u.Quantity(np.nan, u.m)
+                    dl1_container.length = u.Quantity(np.nan, u.m)
+                    dl1_container.wl  = u.Quantity(np.nan, u.m)
+
+            for p in parameters_to_update:
```

Comment:
shall we define `parameters_to_update` as a list external to the  `lstchain_mc_dl1ab.py` script?

Reply:
No, they're always the same. And again, we want to move to ctapipe standard DL1 container as soon as possible, no need to improve this now.